Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "offscreen_gl_context"
 license = "MIT / Apache-2.0"
-version = "0.12.1"
+version = "0.12.2"
```

Comment:
We should do a minor version bump, since we use generated bindings in the public API for this crate.